nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/HandleHttpResponse.java

```diff
@@ -204,7 +206,7 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
 
         try {
             contextMap.complete(contextIdentifier);
-        } catch (final IllegalStateException ise) {
+        } catch (final RuntimeException ise) {
```

Comment:
`ise` seems to be a leftover from the previous version.
Could you please use `ce` here too like in line 197.

Reply:
Renamed to `ce` as requested.